documentation/modules/managing/proc-setting-broker-limits.adoc

```diff
@@ -7,24 +7,44 @@
-You can set a byte-rate threshold and storage quotas to put limits on the clients interacting with your brokers.
+This procedure describes how to set throughput and storage limits on brokers in your Kafka cluster.
+Enable a quota plugin and configure limits using `quotas` properties in the `Kafka` resource.
 
-NOTE: Only one quota plugin can be used in Kafka. 
-By enabling this plugin, the built-in Kafka quotas plugin is automatically disabled.
-This means that you won't see per-client quota metrics, but only aggregated quota metrics.
+.Quota plugins
 
-You can set byte-rate thresholds for producer and consumer bandwidth.
-The total limit is distributed across all clients accessing the broker.
-For example, you can set a byte-rate threshold of 40 MBps for producers.
-If two producers are running, they are each limited to a throughput of 20 MBps.
+There are two types of quota plugins available:
 
-Storage quotas enforce the throttling of Kafka producers based on Kafka disk storage utilization when it reaches the specified limit.
-You can specify the limit in bytes or percentage of available disk space.
-The limit applies to every disk individually.
-It prevents disks filling up too quickly and exceeding their capacity.
-Full disks can lead to issues that are hard to rectify.
+* The `strimzi` type enables the _Kafka Static Quota_ plugin.
+* The `kafka` type enables the built-in Kafka plugin. 
+
+Only one quota plugin can be enabled at a time. 
+The built-in `kafka` plugin is enabled by default.
+Enabling the `strimzi` plugin automatically disables the built-in plugin.
+
+.`strimzi` plugin
+
+The `strimzi` plugin provides storage utilization quotas and dynamic distribution of throughput limits.
+
+* Storage quotas throttle Kafka producers based on disk storage utilization. 
+Limits can be specified in bytes (`minAvailableBytesPerVolume`) or percentage (`minAvailableRatioPerVolume`) of available disk space, applying to each disk individually. 
+When any broker in the cluster exceeds the configured disk threshold, clients are throttled to prevent disks from filling up too quickly and exceeding capacity.
+* A total throughput limit is distributed dynamically across all clients.
+For example, if you set a 40 MBps producer byte-rate threshold, the distribution across two producers is not static. 
+If one producer is using 10 MBps, the other can use up to 30 MBps.
+* Specific users (clients) can be excluded from the restrictions.
+
+NOTE: With the `strimzi` plugin, you see only aggregated quota metrics, not per-client metrics.
+
+.`kafka` plugin
+
+The `kafka` plugin applies throughput limits on a per-user, per-broker basis and includes additional CPU and operation rate limits.
+
+* Limits are applied per user and per broker. 
+For example, setting a 20 MBps producer byte-rate threshold limits each producer to 20 MBps on a per-broker basis across all connections for that user. 
```

Comment:
I think this would make more sense to how it works? Or maybe be a bit less confusing as to what "each producer" means.

```suggestion
For example, setting a 20 MBps producer byte-rate threshold limits each user to 20 MBps on a per-broker basis across all producer connections for that user. 
```